src/vs/platform/userDataProfile/common/userDataProfile.ts

```diff
@@ -69,8 +69,7 @@ export function isUserDataProfile(thing: unknown): thing is IUserDataProfile {
 
 export const PROFILES_ENABLEMENT_CONFIG = 'workbench.experimental.settingsProfiles.enabled';
 
-export type EmptyWindowWorkspaceIdentifier = 'empty-window';
-export type WorkspaceIdentifier = ISingleFolderWorkspaceIdentifier | IWorkspaceIdentifier | EmptyWindowWorkspaceIdentifier;
+export type WorkspaceIdentifier = ISingleFolderWorkspaceIdentifier | IWorkspaceIdentifier | IEmptyWorkspaceIdentifier;
```

Comment:
Use the `IAnyWorkspaceIdentifier` type here and remove `WorkspaceIdentifier`

Reply:
Sure